tests/spec/fs.truncate.spec.js

```diff
@@ -25,6 +25,21 @@ describe('fs.truncate', function() {
     });
   });
 
+  it('should error when length is NaN', function(done) {
+    var fs = util.fs();
```

Comment:
It maybe naming variables like fsPromises instead of fs() that is make more clear.

Reply:
Nice catch @Janice0218 - I chose not to use `fsPromises` since I'm not using the asynchronous version of it. I should probably reopen a new PR to focus on `fsPromises.truncate` instead 🤔